### ISSUE TITLE
[BUG] Every time I run any query or view a table I'm getting an error message showing in the output

### ISSUE THREAD

**Issue by xela-nosretap** [cweijan/vscode-mysql#480]
Whenever I interact with my DB in any way I keep on getting the output window pop open with these error messages.

![image](https://user-images.githubusercontent.com/931083/154274845-a4d2e8d6-0b31-4e55-b405-77c367761c65.png)

I'm not sure why it's trying to connect to port 3308 when I have 3306 set in my config.

- OS: Machine is running Windows 11, using Debian under WSL2 in VSCode, DB is in a docker container.
- Database Type: MySQL

**Comment by cweijan** [comment]
Do you have a connection to port 3308? This extension does not hardcode 3308.
Is the extension version you are using 4.6.0?

**Comment by xela-nosretap** [comment]
I'm using version 4.6.0 of the extension.

I don't believe I have any connections to port 3308. My docker compose file has no reference to that port, I can't see that port being used in either Windows or Debian.

**Comment by cweijan** [comment]
I need more information:
1. Does this connection currently display the database structure correctly?
2. What do you do by interaction?

**Comment by xela-nosretap** [comment]
I think I have figured out the issue.

My local connection to my database in the docker container uses port 3306.
I also have a connection saved which points to my staging environment, this one usesto port 3308. I need to use run ScaleFT to create a tunnel to access this.

Seems that the reason I've been getting those messages is because I didn't have the tunnel open and the extension was trying to connect to the database because I hadn't manually closed it. 

I wrongly assumed that the connections were closed by default when opening VSCode.
